docker/Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN apt-get update \
 # Install the Google Cloud SDK
 ENV HOME /
 ENV CLOUDSDK_PYTHON_SITEPACKAGES 1
-RUN wget -O google-cloud-sdk.tar.gz https://storage.googleapis.com/cloud-sdk-release/google-cloud-sdk-218.0.0-linux-x86.tar.gz \
+RUN curl https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz > google-cloud-sdk.tar.gz \
```

Comment:
is curl > wget? just wondering about the change

Reply:
No particular reason. Some example from google using curl.